theano/tensor/blas.py

```diff
@@ -250,10 +270,11 @@ def perform(self, node, inputs, out_storage):
             out = numpy.dot(A, x)
             if alpha != 1:
                 out *= alpha
-            if beta != 1:
-                out += beta * y
-            else:
-                out += y
+            if beta != 0:
+                if beta != 1:
+                    out += beta * y
+                else:
+                    out += y
```

Comment:
IRL discussion with @abergeron This duplicate the BLAS interface that is beta is 0, the multiplication isn't done. So this is consistent with the c code. So this is good.